mapbook-iOS.xcodeproj/project.pbxproj

```diff
@@ -242,8 +242,8 @@
 				3EDC84421F1E8B94002DBCF2 /* Sources */,
 				3EDC84431F1E8B94002DBCF2 /* Frameworks */,
 				3EDC84441F1E8B94002DBCF2 /* Resources */,
-				94AB9D2F225E8B8C0040D989 /* ShellScript */,
 				94AB9D33225E8C090040D989 /* Embed Frameworks */,
+				94AB9D2F225E8B8C0040D989 /* ShellScript */,
```

Comment:
It would be nice to rename the build phase to something more descriptive. We used "Strip Framework" (after the name of the script) in iOS sample viewer.

Reply:
I agree that it could be more clear, but was trying to be consistent with the existing open-source apps.

I have pushed [5d5cbba](https://github.com/Esri/mapbook-ios/pull/87/commits/5d5cbba2f5132c8bcb736ef015eddfc9688f7b8c) to address this request.